test/integration/base_integration_test.cc

```diff
@@ -302,7 +302,7 @@ void BaseIntegrationTest::createGeneratedApiTestServer(
 
     // Wait for listeners to be created before invoking registerTestServerPorts() below, as that
     // needs to know about the bound listener ports.
-    Event::TestTimeSystem::RealTimeBound bound(TestUtility::DefaultTimeout);
+    Event::TestTimeSystem::RealTimeBound bound(2 * TestUtility::DefaultTimeout);
```

Comment:
Do you mind adding a comment inline about the doubling?

Reply:
Added.